targets/targets.json

```diff
@@ -3593,7 +3595,7 @@
     "NRF52_DK": {
         "supported_form_factors": ["ARDUINO"],
         "inherits": ["MCU_NRF52832"],
-        "release_versions": [],
+        "release_versions": ["2", "5"],
```

Comment:
We don't claim mbed 2 support for the nrf52 series.

Reply:
fixed